docs/design/supply/network-import-tned.md

```diff
@@ -1,3 +1,113 @@
+
+## Network Files
+
+The ABM3 model system has been configured to be compatible with SANDAG's Transportation Network Editing Database (TNED) system, which is utilized to edit, maintain and generate transportation networks. The TNED network files, generated via an ETL (i.e., Extract, Tranform, Load) procedure, serve as inputs to the ABM3 model system's import network procedure and are produced in text file, shapefile, geodatabase table and geodatabase feature class geodatabase formats. There are, additionally, some non-TNED input network files which are manually maintained.
+
+The following are the required network files used during the Emme import network procedure:
+
+<table>
+  <tr>
+    <th>File</th>
+    <th>Source</h>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>EMMEOutputs.gdb/TNED_HwyNet</td>
+    <td>TNED</td>
+    <td>Roadway network links</td>
+  </tr>
+  <tr>
+    <td>EMMEOutputs.gdb/TNED_HwyNodes</td>
+    <td>TNED</td>
+    <td>Roadway network nodes</td>
+  </tr>
+  <tr>
+    <td>EMMEOutputs.gdb/TNED_RailNet</td>
+    <td>TNED</td>
+    <td>Rail network links</td>
+  </tr>
+  <tr>
+    <td>EMMEOutputs.gdb/TNED_RailNodes</td>
+    <td>TNED</td>
+    <td>Rail network nodes</td>
+  </tr>
+  <tr>
+    <td>EMMEOutputs.gdb/Turns</td>
+    <td>TNED</td>
+    <td>Turn prohibition records</td>
+  </tr>
+  <tr>
+    <td>special_fares.txt</td>
+    <td>Manually Maintained</td>
+    <td>Special fares in terms of boarding and incremental in-vehicle costs</td>
+  </tr>
+  <tr>
+    <td>timexfer_{time_of_day}.csv</td>
+    <td>Manually Maintained</td>
+    <td>Timed transfer pairs of lines, by period. Where <i>time_of_day</i> refers to EA, AM, MD, PM, or EV.</td>
+  </tr>
+  <tr>
+    <td>trrt.csv</td>
+    <td>TNED</td>
+    <td>Attribute data (modes, headways) for the transit lines</td>
+  </tr>
+  <tr>
+    <td>trlink.csv</td>
+    <td>TNED</td>
+    <td>Sequence of links (routing) for the transit lines</td>
+  </tr>
+  <tr>
+    <td>trstop.csv</td>
+    <td>TNED</td>
+    <td>Stop data for the transit lines</td>
+  </tr>
+  <tr>
+    <td>MODE5TOD.csv</td>
+    <td>Manually Maintained</td>
+    <td>Global (per-mode) transit cost and perception attributes</td>
+  </tr>
+  <tr>
+    <td>vehicle_class_toll_factors.csv</td>
+    <td>Manually Maintained</td>
+    <td>Factors to adjust the toll cost by facility name and class</td>
+  </tr>
+</table>
+
+## Import Network Procedure
+
+This section describes the main steps carried out during the Emme import network procedure. The entire process is executed by the [import_network.py](https://github.com/SANDAG/ABM/blob/ABM3_develop/src/main/emme/toolbox/import/import_network.py) script. The descriptions below are excerpts and slight adaptations from the [User Guide - SANDAG Travel Model in Emme](https://github.com/SANDAG/ABM/wiki/files/user_guide_sandag_emme.pdf) report.
```

Comment:
Please remove reference to the old wiki. We want folks to flow in the other direction (come from old wiki to this place). This sentence about user guide is not needed here at this time. We might get the files and reports from the old wiki to this site in the future.

Reply:
@bhargavasana done. Removed reference to ABM2+ Wiki Emme report.